client/lv_utils.py

```diff
@@ -44,7 +44,8 @@
 
 @error.context_aware
 def vg_ramdisk(vg_name, ramdisk_vg_size,
-               ramdisk_basedir, ramdisk_sparse_filename):
+               ramdisk_basedir, ramdisk_sparse_filename,
+               use_ram=True):
```

Comment:
Sure. I think `use_ramdisk` fits the local vocabulary better but you have a point as well - I will use tmpfs then.